src/Animated.js

```diff
@@ -45,6 +46,9 @@ const Animated = {
   // configuration
   addWhitelistedNativeProps,
   addWhitelistedUIProps,
+
+  // nodes with custom cache
+  animatedArrayFrom,
```

Comment:
One more thing! Sorry for troubles and thanks for your work!
```suggestion
  arrayFrom,
```

Reply:
😱 🙀 😱